frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -1,27 +1,32 @@
 <script setup lang="ts">
-import { onUnmounted, shallowRef, type PropType } from 'vue';
+import { onUnmounted, shallowRef, computed, type PropType } from 'vue';
 import Dropdown, { type DropdownChangeEvent } from 'primevue/dropdown';
 import TabView from 'primevue/tabview';
 import TabPanel from 'primevue/tabpanel';
 import ManagePermissionsTitle from '@/components/managePermissions/ManagePermissionsTitle.vue';
-import UserDataTable from '@/components/managePermissions/UserDataTable.vue';
+import UserDataTable from '@/components/managePermissions/table/UserDataTable.vue';
+import ApplicationAdminTable from './table/ApplicationAdminTable.vue';
```

Comment:
Hi Jota, could we make the import in a consistent way? thanks!!

Reply:
Hello, Catherine! Just did, thanks!